pkg/thing/mocks/thing_proxy.go

```diff
@@ -0,0 +1,32 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/thing/delivery/http"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakeThingProxy struct {
+	mock.Mock
+	ReturnErr error
+}
+
+func (ftp *FakeThingProxy) Create(id, name, authorization string) (idGenerated string, err error) {
+	ret := ftp.Called(id, name, authorization)
+	return ret.String(0), ret.Error(1)
+}
+
+func (ftp *FakeThingProxy) UpdateSchema(authorization, thingID string, schema []entities.Schema) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method FakeThingProxy.UpdateSchema should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)